addons/website/static/src/js/content/menu.js

```diff
@@ -336,12 +336,14 @@ publicWidget.registry.FadeOutHeader = BaseDisappearingHeader.extend({
      * @override
      */
     _hideHeader: function () {
+        this.$el.addClass('o_header_hidden');
```

Comment:
Why do you need a class? Can't you save that in a variable instead?

Reply:
yes